chopper.py

```diff
@@ -229,27 +249,31 @@ def split_by_columns(filepath: Path, col_list: list[str], config: Config) -> lis
     for file in files.values():
         file["fout"].close()
 
-    if not config.is_original:
-        os.remove(filepath)
+    if filepath.absolute().parents[0] == config.output_directory.absolute():
+        filepath.unlink()
```

Comment:
You'll see this pattern a few times. The check is cheap, so why bother trying to store the state and think about all the edge conditions?